json/signin.js

```diff
@@ -0,0 +1,70 @@
+const emailInput = document.querySelector("#email");
+const passwordInput = document.querySelector("#password");
+const confirmBtn = document.querySelector('#confirm-button');
+const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+
+/*email 형식인지 체크해주는 함수*/
+function emailCheck(email_address){     
+	if (!email_regex.test(email_address)) { 
+		return false; 
+	} else {
+		return true;
+	}
+}
```

Comment:
함수명이 더 자세할 필요가 있을 것 같습니다. 
어떤 것을 체크하는지, validCheck인지 invalidCheck인지 더 명확하게 네이밍해주셔야 return값을 예측하기 쉬워집니다.

```suggestion
function emailValidCheck(email_address){     
	return email_regex.test(email_address)
}
```